Traceback (most recent call last):
  File "D_fake.py", line 27, in <module>
    img_fake = np.array(img_fake, dtype='float32')
ValueError: setting an array element with a sequence.
